core-services/prow/02_config/_config.yaml

```diff
@@ -2231,6 +2228,208 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - opendatahub-io/odh-manifests
+  - includedBranches:
+    - release-4.5
+    labels:
+    - group-lead-approved
+    - lgtm
+    - approved
+    - bugzilla/valid-bug
```

Comment:
Just to be sure, `cherry-pick-approved` is not required intentionally?

Reply:
Yes. We will enable cherry-pick-approved when the GA comes.